### PR TITLE
Add `satellite` image to `skimage.data`

### PR DESCRIPTION
**Pull request by xwang2**
A satellite imagery was downloaded.

### CODE REVIEW

File: skimage/data/__init__.py [hunk 1]
```diff
@@ -327,6 +328,23 @@ def rocket():
...
     return load("rocket.jpg")
 
+def satellite():
+    """Launch photo taken by satellite.
+
+    This is a satellite imagery showing an urban area
+
+    Notes
+    -----
+    This image was downloaded from USGS public data portal https://earthexplorer.usgs.gov/
```
Comment (stefanv) [inline skimage/data/__init__.py]: It is fine to include the copyright info link here, but it should explicitly be stated that these images are in the public domain.

**Comment by stefanv** [comment]
I'll let @emmanuelle do a round of reviewing too.

File: skimage/data/__init__.py [hunk 2]
```diff
@@ -328,6 +329,31 @@ def rocket():
...
+    https://www2.usgs.gov/laws/info_policies.html
+
+    Courtesy of U.S. Geological Survey
+    Department of the Interior/USGS
+
+    Product name: High Resolution Orthoimagery
+    Acquisition Date: 27-Oct-14
+
+    Returns
+    -------
```
Comment (emmanuelle) [inline skimage/data/__init__.py]: Could you please add something after Returns, as in the other functions of the module?

**Comment by emmanuelle** [comment]
Thanks for the image! Do you intend to submit the example for the gallery in this PR or in a separate one?

File: skimage/data/__init__.py (hunk 2 above)
Comment (xwang2) [inline skimage/data/__init__.py]: Comments added after Returns.

**Comment by xwang2** [comment]
Closed.

**Pull request closed without merging**